.github/workflows/fetch.yml

```diff
@@ -30,7 +31,7 @@ jobs:
       - name: Build the Docker image
         run: docker build . --file Dockerfile --tag juscaba
       - name: Run the container
-        run: docker run -v $(pwd):/tmp/juscaba juscaba ./run.sh $EXPEDIENTES
+        run: docker run -v $(pwd):/tmp/juscaba --env "DOCUMENTOS_URL=${DOCUMENTOS_URL}" juscaba ./run.sh $EXPEDIENTES
```

Comment:
do not mix env and args

Reply:
now that's just mean, one is a config, the other indicates which actions to take

Reply:
Ermm.. then maybe you want a config file ?
Still looks off...